src/loaders/ccloudResourceLoader.test.ts

```diff
@@ -722,6 +722,35 @@ describe("CCloudResourceLoader", () => {
     });
   });
 
+  describe("loadStatementsForProviderRegion", () => {
+    let flinkArtifactsApiStub: sinon.SinonStubbedInstance<FlinkArtifactsArtifactV1Api>;
+    beforeEach(() => {
```

Comment:
wanted to move this to the top level describe block because there's some repeated code, but it caused a bunch of other tests in here to fail